src/app/components/Item/ItemHoldings.jsx

```diff
@@ -138,7 +138,9 @@ class ItemHoldings extends React.Component {
 
     return (
       <div id="item-holdings" className="item-holdings">
-        <h2>{this.props.title}</h2>
+        <dt className="list-multi-control">
```

Comment:
Okay, I see. In `getTable`, you can remove the surrounding `<dl>` wrapper and this `dt` will have definition.